Assets/SEE/Game/Operator/EdgeOperator.cs

```diff
@@ -32,6 +32,9 @@ public partial class EdgeOperator : GraphElementOperator<(Color start, Color end
         /// </summary>
         private SEESpline spline;
 
+        /// Shader property that enables or disables the edge direction (data flow) animation.
```

Comment:
Missing summary tags:
```suggestion
        /// <summary>
        /// Shader property that enables or disables the edge direction (data flow) animation.
        /// </summary>
```